meta-resin-common/recipes-connectivity/modemmanager/modemmanager_%.bbappend

```diff
@@ -9,14 +9,13 @@ SRC_URI_append = " \
 "
 
 do_install_append() {
-    install -d ${D}/etc/udev/rules.d/
     install -d ${D}/lib/udev/
```

Comment:
Shouldn't you also make the rules.d directory?
 `install -d ${D}/lib/udev/rules.d/`